earthpy/plot.py

```diff
@@ -149,11 +163,22 @@ def plot_bands(
         axs_ravel = axs.ravel()
         for ax, i in zip(axs_ravel, range(total_layers)):
             band = i + 1
-            ax.imshow(es.bytescale(arr[i]), cmap=cmap)
+            if scale:
+                arr_im = es.bytescale(arr[i])
+            else:
+                arr_im = arr[i]
+            if len(vmin_vmax) == 2:
+                im = ax.imshow(
+                    arr_im, cmap=cmap, vmin=vmin_vmax[0], vmax=vmin_vmax[1]
```

Comment:
Does the `extent` argument need to get passed in here, or is it not relevant in these cases?

Reply:
i think if we set it to `None` it should just be called in the imshow() method.

Reply:
it's been added!

Reply:
Cool - I opened an issue here after noticing that we weren't testing that the extent parameter does anything (but we can deal with this in another PR): https://github.com/earthlab/earthpy/issues/281